src/components/TopBar/AboutDialog.js

```diff
@@ -377,26 +378,44 @@ const Module = ({ type, name, version, license }) => {
                 },
             }}
         >
-            <Stack
-                direction="row"
-                justifyContent="flex-start"
-                alignItems="baseline"
-                spacing={1}
-            >
-                {ModuleTypesIcons[type] || ModuleTypesIcons['other']}
-                <Typography display="inline-block" noWrap>
-                    {name || '<?>'}
-                </Typography>
-                <Typography
-                    variant="caption"
-                    color={(theme) => theme.palette.text.secondary}
-                    display="inline"
-                    marginLeft={1}
-                    noWrap
+            <Tooltip title={(name || '<?>') + ' ' + (version || '')}>
+                <Stack
+                    direction="row"
+                    justifyContent="flex-start"
+                    alignItems="baseline"
+                    spacing={1}
                 >
-                    {version || null}
-                </Typography>
-            </Stack>
+                    <Box
```

Comment:
Give the `sx` prop to `ModuleTypeIcons` directly instead of creating a `Box`.

ex :
```
const ModuleTypesIcons = (
    sx = {
        flexGrow: 0,
        position: 'relative',
        top: '4px',
        flexShrink: 0,
    }
) => ({
    app: <WidgetsOutlined sx={sx} fontSize="small" color="primary" />,
    server: <DnsOutlined sx={sx} fontSize="small" color="secondary" />,
    other: <QuestionMark sx={sx} fontSize="small" />,
});
```

then

`{ModuleTypesIcons()[type] || ModuleTypesIcons()['other']}`

Reply:
or you can just do this I think it's better

```
const iconsStyle = {
    flexGrow: 0,
    position: 'relative',
    top: '4px',
    flexShrink: 0,
};

const ModuleTypesIcons = {
    app: <WidgetsOutlined sx={iconsStyle} fontSize="small" color="primary" />,
    server: <DnsOutlined sx={iconsStyle} fontSize="small" color="secondary" />,
    other: <QuestionMark sx={iconsStyle} fontSize="small" />,
};
```

Reply:
Yes, I like the second version better.